src/core_io.h

```diff
@@ -46,6 +46,6 @@ std::string EncodeHexTx(const CTransaction& tx, const int serializeFlags = 0);
 std::string SighashToStr(unsigned char sighash_type);
 void ScriptPubKeyToUniv(const CScript& scriptPubKey, UniValue& out, bool fIncludeHex);
 void ScriptToUniv(const CScript& script, UniValue& out, bool include_address);
-void TxToUniv(const CTransaction& tx, const uint256& hashBlock, UniValue& entry, bool include_hex = true, int serialize_flags = 0, const CTxUndo* txundo = nullptr);
+void TxToUniv(const CTransaction& tx, const uint256& hashBlock, UniValue& entry, bool include_hex = true, int serialize_flags = 0, const CTxUndo* txundo = nullptr, const bool display_prev_out = false);
```

Comment:
Can we just use `BlockToJsonVerbosity` here? (Maybe rename it)

Reply:
I renamed it to TxVerbosity, and moved it to src/core_io.h. Feels cleaner than passing bools.

Reply:
(Please mark as resolved 🙏 )

Reply:
(Please mark as resolved 🙏 )